tests/checkEntriesFunctionTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+require ('../checkEntriesFunction.php');
+
+use PHPUnit\Framework\TestCase;
+
+class FunctionTest extends TestCase
```

Comment:
You might consider using "class CheckEntriesTest extends TestCase" instead. This is because having all of your tests use "class FunctionTest extends TestCase" means that each of your unit tests must be run one at a time. Ash picked me up on this in my code review.